en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -112,6 +112,20 @@ This section demonstrates how to create a rule for specifying data to be forward
      "t/#"
    ```
 
+   If you're using the MQTTv5 protocol, the publish properties are forwarded to the remote broker as-is, provided that the `pub_props` field is included in the rule SQL. This is the case in the `SELECT * FROM t/#` example shown above.
+
+   To dynamically add more user properties, you can include them in the `pub_props` field that the rule outputs. For example, the following rule will add a user property with key and value taken from the incoming payload:
+
+   ```sql
+   SELECT
+     *,
+     map_put(concat('User-Property.', payload.extra_key), payload.extra_value, pub_props) as pub_props
+   FROM
+     't/#'
+   ```
+
+   This ensures that custom user properties from the payload are included in the MQTT message sent to the remote broker.
```

Comment:
I think this might be confusing.  As the paragraph above states, what ensures that user properties are forwarded is including `pub_props` in the select statement.

Reply:
I just removed this sentence because it sounds repetitive.